ludwig/schema/features/utils.py

```diff
@@ -70,14 +88,14 @@ def get_input_feature_conds():
 
 
 @DeveloperAPI
-def get_output_feature_jsonschema():
+def get_output_feature_jsonschema(model_type: str = MODEL_ECD):
```

Comment:
nit: Remove the default argument and make users pass the model type explicitly.

Side reading: https://quuxplusone.github.io/blog/2020/04/18/default-function-arguments-are-the-devil/#conclusion

Reply:
fixed and read